pallets/spin-polkadot/src/lib.rs

```diff
@@ -0,0 +1,257 @@
+//! # SPIN Pallet
```

Comment:
Please provide at least a one sentence description for the crate.

Reply:
I've add overview of the crate